index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>Cooperative Scheduling of Background Tasks</title>
-  <script src='//www.w3.org/Tools/respec/respec-w3c-common' async class=
+  <script src='https://www.w3.org/Tools/respec/respec-w3c-common' async class=
```

Comment:
Just checking that this change is intended.

Reply:
Yeah, I changed it to be able to test the spec locally without spinning up a web server. Not strictly related to this change though, so I can spin it to a separate PR if you think it's necessary